lib/ssm_storage/ssm_storage_db.rb

```diff
@@ -0,0 +1,76 @@
+class SsmStorageDb
```

Comment:
This can be
```suggestion
module SsmStorage
class Db
```
Then you'd refer to them as `SsmStorage::Db`. It reflects the folder structure better for rails applications. I'm afraid otherwise we will run into a lot of 'class not found'. This might also remove the requirement to explicitly require these files.